backend/apps/matching/models/user.py

```diff
@@ -49,11 +49,16 @@ class Meta(AbstractUser.Meta):
             # we can only have either participant a or participant b for one user
             models.CheckConstraint(
                 name="%(app_label)s_%(class)s_has_only_one_participant",
-                check=(~models.Q(is_A=False, is_B=False,)),
+                check=(~(models.Q(is_A=True) & models.Q(is_B=True))),
             ),
-            # only either staff or participant
+            # only either staff or participant or none
```

Comment:
Is `none` sensible?

Reply:
That is a design choice - I just wanted to reflect what the test is currently testing for clarification. Design by @feeds , but I can imagine that these validations will still change over the upcoming weeks so I wouldn't think too much about it now. We should probably have another design-choices call soon, there are some more questions pending.